### PR TITLE
Generate reportdb documentation 

### PR DESCRIPTION
**Pull request by mackdk**
## What does this PR change?

This PR adds the comments and the script needed to generate the reporting database documentation. It is composed of three main parts:

- The SQL script to add the comments and prepare the database for the analysis. These are located in the directory `schema/reportdb/postgresql/docs` as this script are PostgreSQL specific.
- The support files needed to generated the documentation, located in the `schema/reportdb/docs` directory. There are the java libs for schemaspy and the postgresql jdbc driver, the html template, the fonts and the property file with the schemaspy settings
- The scripts to execute the process in a docker container located in `susemanager-utils/testing`

## GUI diff

No difference.

- [X] **DONE**

## Documentation

Documentation is implemented through SQL comments and foreign keys added to the database and generated using the SchemaSpy tool which is bundled inside the `schema/reportdb/docs` directory. After the script are executed the database is built updated and examined in a docker container and the result of the operation is the HTML documentation.

- [X] **DONE**

## Test coverage
- No tests: No code changes

- [X] **DONE**

## Links

Fixes SUSE/spacewalk/issues/16564

- [X] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [X] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

**Comment by mackdk** [comment]
Continuation of https://github.com/uyuni-project/uyuni/pull/5233

File: schema/reportdb/postgres/docs/tables/ChannelPackage.pre [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+COMMENT ON COLUMN ChannelPackage.package_version
+  IS 'The package version';
+COMMENT ON COLUMN ChannelPackage.package_release
+  IS 'The package release number';
+COMMENT ON COLUMN ChannelPackage.package_type
+  IS 'The type of the package. Possible values: rpm, deb';
+COMMENT ON COLUMN ChannelPackage.package_arch
+  IS 'The package architecture';
+COMMENT ON COLUMN ChannelPackage.synced_date
+  IS 'The timestamp of when this data was last refreshed.';
```
Comment (mcalmer) [inline schema/reportdb/postgres/docs/tables/ChannelPackage.pre:36]:
```suggestion
COMMENT ON COLUMN ChannelPackage.synced_date
  IS 'The timestamp of when this data was last refreshed.';
```
Remove dropped columns

**Review by mcalmer — CHANGES_REQUESTED** [review]
One change needed for the latest changes in the ChannelPackage table

File: susemanager-utils/testing/docker/scripts/generate-reportdb-docs.sh [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    rm -f $OUTPUT_FILE
+fi
+
+OUTPUT_DIR=$(cat $CONFIG_FILE | grep schemaspy.o | cut -c 13-)
+if [ -d $OUTPUT_DIR ]; then
+    echo Removing previous generated docs at $OUTPUT_DIR
+    rm -rf $OUTPUT_DIR
+fi
+
+java -cp $(build-classpath ongres-scram) -jar /root/schemaspy.jar -configFile $CONFIG_FILE -dp $(build-classpath postgresql-jdbc)
```
Comment (mcalmer) [inline susemanager-utils/testing/docker/scripts/generate-reportdb-docs.sh]: The download of `schemaspy.jar` happens not here?

**Review by mcalmer — APPROVED** [review]

**Pull request merged**